fetch/api/basic/request-upload.any.js

```diff
@@ -123,3 +125,27 @@ testUploadFailure("Fetch with POST with ReadableStream containing Blob", url,
       controller.close();
     }})
   });
+
+promise_test(async (test) => {
+  const resp = await fetch(
+    "/fetch/connection-pool/resources/network-partition-key.py?"
+    + `status=421&uuid=${token()}&partition_id=${get_host_info().ORIGIN}`
+    + `&dispatch=check_partition&addcounter=true`,
+    {method: "POST", body: "foobar"});
+  assert_equals(resp.status, 421);
```

Comment:
I don't understand this. Does this mean there's no recursion and user agents will only do it once?

Reply:
Updated server message. There was only one recursion.

Reply:
But why is the response status still 421?

Reply:
It means
 server responded twice and the browser gave up resend.

Reply:
I see, but it's also conforming to do another request and open another connection, right? If we want the recursion limit to be 1, we should just put that in Fetch.

Reply:
I don't have strong opinion to the recursion limit.
Updated the test to check if request should be retried at least once and
connection should be created as same as request.

Reply:
Thanks, I have asked for some feedback from the HTTP community. I'm happy to help you finish this once we get some feedback. I suspect that if returning 421 is okay here, it might also be okay when using a stream.

Reply:
Following https://github.com/whatwg/fetch/pull/1141#issuecomment-789569885, updated both tests.